hack/update-codegen.sh

```diff
@@ -17,6 +17,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+GOPATH="${GOPATH:-"${HOME}/go"}"
```

Comment:
`go env GOPATH` is probably a better idea if we're just using the `go` in `$PATH` anyhow.